.coveragerc

```diff
@@ -1,4 +1,4 @@
 [run]
-concurrency = threading
+concurrency = thread
```

Comment:
Please change this to `threading` before merging.

Reply:
Sure.